sequencing/vcf_slice/dxapp.json

```diff
@@ -1,69 +1,90 @@
 {
   "name": "vcf_slice",
   "title": "Slice VCF File(s)",
-  "summary": "vcf_slice",
+  "summary": "Return a small section of a VCF file or files (similar to tabix).",
   "dxapi": "1.0.0",
-  "version": "0.0.1",
+  "version": "0.1.0",
   "categories": [
     "Variation Calling"
   ],
   "inputSpec": [
     {
       "name": "vcf_fn",
-      "label": "VCF Files",
+      "label": "VCF files",
+      "help": "Variant files to be sliced, in VCF format.",
       "class": "array:file",
       "optional": false,
-      "patterns" : ["*.vcf.gz"]
+      "patterns": ["*.vcf.gz"]
     },
     {
       "name": "vcfidx_fn",
-      "label": "VCF Index Files",
+      "label": "VCF index files",
```

Comment:
"vcfidx_fn" -> "variants_vcfgztbis",
If input is a \*.vcf.gz file, then the index filename should be \*.vcf.tbi.  (Likely the case)
If input is a \*.vcf file, then the index filename should be \*.vcf.idx.

Also need to update variables accordingly in the app script.

Reply:
A common filename for *.vcf.gz file is *.vcf.gz.tbi